lib/go-tc/deliveryservices.go

```diff
@@ -245,7 +245,8 @@ type DeliveryServiceV40 struct {
 
 	// TLSVersions is the list of explicitly supported TLS versions for cache
 	// servers serving the Delivery Service's content.
-	TLSVersions []string `json:"tlsVersions" db:"tls_versions"`
+	TLSVersions       []string               `json:"tlsVersions" db:"tls_versions"`
+	GeoLimitCountries *GeoLimitCountriesType `json:"geoLimitCountries"`
```

Comment:
`GeoLimitCountriesType` is already a reference type - `nil` is a valid value for a `[]string` - so I don't think this needs to be a pointer.